src/DonationForms/Blocks/DonationFormBlock/resources/app/index.tsx

```diff
@@ -73,6 +73,7 @@ function DonationFormBlockApp({
             id={embedId}
             src={dataSrc}
             checkOrigin={false}
+            heightCalculationMethod={'taggedElement'}
```

Comment:
I revisited the [original embed file](https://github.com/impress-org/givewp/blob/a52ebdd4da22bbbffbe6fc1ca84c74bbba3a76ee/src/DonationForms/resources/embed.ts#L3) and remembered that I added the iframe resizer to look for the data attribute `data-givewp-embed`.  Is that still at thing?

Reply:
After looking a bit further , I found this is still very much a thing 😄 . 

`embed.ts` is required for `[give_receipt]` which is important when admins opt to use the Confirmation Page Redirect setting. Without it the v3 confirmation page does **not** size properly.

In the console I can see the iframe using `data-givewp-embed`.
<img width="1888" alt="Screenshot 2024-11-06 at 2 30 59 PM" src="https://github.com/user-attachments/assets/798be4a1-83ec-4ec3-90e6-3a328592cac5">

Reply:
Okay so if i'm understanding correctly, we don't need the embed script for the donation form because we are using the react version, but the confirmation page does not use that so we still need the vanilla embed script.  That sound right?

Reply:
Yes, the confirmation page that displays through the regular form process still uses the React version. Confirmation pages rendered through other avenues, for example shortcodes will still require the embed file.